yarn.lock

```diff
@@ -19133,6 +19133,11 @@ reselect@^4.1.7:
   resolved "https://registry.yarnpkg.com/reselect/-/reselect-4.1.7.tgz#56480d9ff3d3188970ee2b76527bd94a95567a42"
   integrity sha512-Zu1xbUt3/OPwsXL46hvOOoQrap2azE7ZQbokq61BQfiXvhewsKDwhMeZjTX9sX0nvw1t/U5Audyn1I9P/m9z0A==
 
+reselect@^5.1.0:
```

Comment:
reselect is already a dependency of rtk, do you think it would make sense to try and use the same version?

Reply:
Version 4 doesn't export lru cache